content/client-lib-development-guide/features.textile

```diff
@@ -1730,11 +1734,19 @@ class RealtimeChannel:
   publish(name: String?, data: Data?) => io // RTL6i
   subscribe((Message) ->) => io // RTL7a
   subscribe(String, (Message) ->) => io // RTL7b
+  subscribe(MessageFilterObject, (Message) ->) // RTL22
   unsubscribe() // RTL8a, RTE5
   unsubscribe((Message) ->) // RTL8a
   unsubscribe(String, (Message) ->) // RTL8a
+  unsubscribe(MessageFilterObject, (Message) ->) // RTL22
   setOptions(options: ChannelOptions) => io // RTL16
 
+class MessageFilterObject:
```

Comment:
Could this just be `MessageFilter`?

Reply:
[Done](https://github.com/ably/docs/pull/1444/commits/9bbc7c471ce1828decf3127b4a48f3275388e747)